localization.md

```diff
@@ -1,22 +1,33 @@
+
+Turn instructions are primarily designed to be announced by either the Mapbox Voice API (powered by [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/SupportedLanguage.html)) or [TextToSpeech](https://developer.android.com/reference/android/speech/tts/TextToSpeech). By default, this SDK uses the Mapbox Voice API, which requires an Internet connection at various points along the route. If the Voice API lacks support for the turn instruction language or there is no Internet connection, TextToSpeech announces the instructions instead. 
+
+By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling. To override the measurement system used in spoken instructions, set the `MapboxNavigationOptions.Builder#unitType` property when calculating the route with which to start navigation.
+
+The upcoming road or ramp destination is named according to the local or national language. In some regions, the name may be given in multiple languages.
+
 ## Supported languages
 
-| Language   | User interface | Mapbox Voice API | Android TextToSpeech
-|------------|:--------------:|:----------------:|:---------------------:
-| Catalan    | -              | —                | —
-| Chinese    | -              | ✅               | ✅
-| Danish     | -              | ✅               | ✅
-| Dutch      | -              | ✅               | ✅
-| English    | ✅             | ✅               | ✅
-| French     | -              | ✅               | ✅
-| German     | -              | ✅               | ✅
-| Hebrew     | -              | -                | —
-| Hungarian  | -              | —                | ✅
-| Italian    | -              | ✅               | ✅
-| Portuguese | -              | ✅               | ✅
-| Polish     | —              | ✅               | ✅
-| Romanian   | —              | ✅               | ✅          
-| Russian    | -              | ✅               | ✅
-| Spanish    | -              | ✅               | ✅
-| Swedish    | -              | ✅               | ✅
-| Turkish    | —              | ✅               | ✅
-| Vietnamese | -              | ✅               | ✅
+The table below lists the languages that are supported for user interface elements and for spoken instructions.
+
+| Language   | User interface | [Spoken instructions][osrmti] | Remarks
+|------------|:--------------:|:-----------------------------:|--------
+| Bengali    | ✅             | — 
+| Czech      | ✅             | -
+| Danish     | ✅             | ✅
+| French     | ✅             | ✅
+| English    | ✅             | ✅
+| French     | ✅             | ✅
+| German     | ✅             | ✅
+| Hebrew     | ✅             | - 
```

Comment:
> I gave a `-` to spoken instructions, where iOS is marked as "manual".

“Manual” refers to the fact that the iOS navigation SDK won’t automatically use a language if the UI isn’t already in that language. But my understanding is that the Android navigation SDK will, for instance, deliver Turkish instructions even though the UI is unavailable in Turkish. So if we were to draw parallels between Android and iOS, it would look like this:

iOS | Android
----|----
Manual | ✔️
Uses VoiceOver | Depends on the device; may require third-party text-to-speech
Requires third-party text-to-speech | Depends on the device; may require third-party text-to-speech

> Should I remark that these languages will immediately use `TextToSpeech` and if the speech engine for the given `Locale` is not installed on the device, pronunciation will fail?

Yes. I think we need a way to distinguish between these two cases:

* We have no way of providing instructions in the language.
* We can provide instructions in the language, but it’s up to the device or a third-party text-to-speech engine to speak it.

For the first case, — is appropriate. But for the second case, how about a ✅ with a remark about text-to-speech depending on the device. We’d be adding several rows to this table, including Chinese.